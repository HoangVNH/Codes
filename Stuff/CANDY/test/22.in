10
28
DDVVVVVXXXXDDDDVVVVVVVVVXXXV
30
DDVVVVVDDDXXXXVVDXXXXDDDXXXDDD
50
DDXXXXXXXXXXXXXXXXXXDDDDDDDVVDDDXXXXXVVVVVVDVVVVVD
43
VVVVDXVXXXXXDDDDXXXXXDDDXXDDDDDXXXDDDVVVXXX
24
DDDVVVDDDDDDDDDDDDDXXXXD
47
VDDDDDXXXDDDDDDDDDDVXXXXXDDDDXXXXXXXXDDDDDVVVVD
34
VVVDXXXXXXXXXVXXXVVVVVVVVVVDDDDDXX
40
VXXDDDDVDDDDDDVVVVXXDDXDDXXXXXDXXXXXXXXX
46
VVVVVVVVDDDVVVVDDDVVVVVDDDVVVVDVVVXXXXXXXXXXXX
35
DDVVVVXXDDDDXXDXXXXXDDDDDDDDDDVVVVV
