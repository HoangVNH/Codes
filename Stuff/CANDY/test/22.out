27
29
49
42
23
46
32
39
44
33
